planning/planning/src/chronicles/analysis/detrimental_supports.rs

```diff
@@ -100,14 +100,13 @@ pub fn find_useless_supports(pb: &Problem) -> HashSet<CausalSupport> {
         }
     }
 
-    let mut useful_values = HashSet::new(); // TODO: extend with goals
-
-    for ch in &pb.templates {
-        for cond in &ch.chronicle.conditions {
-            let gval = value_of(&cond.state_var.fluent, &cond.state_var.args, cond.value);
-            useful_values.insert(gval);
-        }
-    }
+    let useful_values = pb
```

Comment:
Maybe just add a comment saying that thiese correspond to goals and thus considered useful.